android/java/res/drawable/wallet_swap_radio_button_selector.xml

```diff
@@ -1,5 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
-<selector xmlns:android="http://schemas.android.com/apk/res/android">
```

Comment:
@samartnik if it's unused resource, we can remove it.

Reply:
It's not used at the moment, but we do have it in the commented code in `brave/android/java/res/layout/activity_buy_send_swap.xml`. Is that code not going to be used?